src/SablierV2MerkleStreamerLL.sol

```diff
@@ -99,6 +104,8 @@ contract SablierV2MerkleStreamerLL is
         );
 
         // Log the claim.
-        emit Claim(index, recipient, amount, streamId);
+        unchecked {
+            emit Claim(index, recipient, amount - fee, streamId);
```

Comment:
riiiiiiiiight

In this case, this is an arbitrary business decision. It's up to us if we want to use the term "claim amount" to refer to the raw amount (exclusive of fees) or to the total amount (the one inclusive of fees).

While I can make a good case in favor of either approach, I'm inclined to go with @andreivladbrg's suggestion (i.e. remove the `-fee` subtraction), because:

1. We do not expect to activate the protocol fee for the vast majority of airstreamed tokens, so the two amounts would match
2. Gas saving - this helps users